spog/ui/crates/backend/src/advisory.rs

```diff
@@ -64,7 +64,7 @@ impl VexService {
     }
 
     pub async fn upload(&self, data: impl Into<Body>) -> Result<String, ApiError> {
-        let url = self.backend.join(Endpoint::Api, "/api/v1/advisory/upload")?;
+        let url = self.backend.join(Endpoint::Vexination, "/api/v1/vex")?;
```

Comment:
I like reusing the API endpoint. Wouldn't that mean that we can remove that in SPoG API?

Reply:
Thanks for reminding it to me. Absolutely, removing unused code is the way. I've just removed them in my last commit